client/components/like-button/index.jsx

```diff
@@ -1,7 +1,8 @@
 /**
  * External Dependencies
  */
-var React = require( 'react/addons' ),
```

Comment:
How did `React.createClass` that work before if `React` was `react/addons` ?
Did `react/addons` contain react as well?